build/tasks/RepoTasks.tasks

```diff
@@ -20,4 +22,15 @@
   <UsingTask TaskName="RepoTasks.ResolveHostingStartupPackages" AssemblyFile="$(_RepoTaskAssembly)" />
   <UsingTask TaskName="RepoTasks.TrimDeps" AssemblyFile="$(_RepoTaskAssembly)" />
   <UsingTask TaskName="RepoTasks.VerifyCoherentVersions" AssemblyFile="$(_RepoTaskAssembly)" />
+
+  <!-- tools from dotnet-buildtools -->
+  <PropertyGroup>
+    <NuGetPackagesPath>$(NUGET_PACKAGES)</NuGetPackagesPath>
+    <NuGetPackagesPath Condition=" '$(NuGetPackagesPath)' == '' AND '$(USERPROFILE)' != '' ">$(USERPROFILE)\.nuget\packages\</NuGetPackagesPath>
```

Comment:
Should UserProfile \ Home be normalized as well?

Reply:
It should be normalized by line 31 (two down)